examples/portfolio-rebalancer/portfolio_rebalancer_agent.py

```
@@ -0,0 +1,105 @@
+    portfolio = Portfolio(
+        id=raw_portfolio.get("id", "unknown"),
+        assets=assets
+    )
+
+    # --- PIPELINE ---
+    drift = analyst.run(portfolio, target)
+
+    symbols = [a.symbol for a in portfolio.assets]
+    prices = pricer.run(symbols)
+
+    risk_scores, portfolio_risk = risk.run(drift)
```

Ruff (0.15.9)

[warning] Unpacked variable `risk_scores` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)